src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
+    Animal.alive.push(this);
```

Comment:
The `hidden` property is only relevant to `Herbivore` instances and should not be defined in the `Animal` class constructor. It's better to remove it from here and handle it exclusively in the `Herbivore` class.